### ISSUE TITLE
Reorganize banner titles

### ISSUE THREAD

**Issue by Roger-luo** [QuEraComputing/bloqade-python#726]
I think we can 

- merge "getting started` especially the guides for using IDEs etc. into the home page because the home page is quite empty now, and you don't get much useful info from it.
- move "Contributing" to the last (before license) because it has less priority.
- the tutorial page should make use of left banners and show the first tutorial directly instead of showing a list of links
- the reference page link is broken: https://queracomputing.github.io/bloqade-python-examples/latest/
- perhaps should write a short post about events in the blog posts in the future (maybe just copy-paste whatever you have for those events cc: @plquera)

lastly, remember to ask people to give a star whenever you talk about the package!

**Comment by weinbe58** [comment]
I don't neccesarily like having the first tutorial pop up as the default page it seems but maybe something using this extension:

https://squidfunk.github.io/mkdocs-material/reference/grids/

Where we have a grid of tutorials with some thumbnails that show what each one is about in a picture.

**Comment by johnzl-777** [comment]
#729 will address the first two points

**Comment by johnzl-777** [comment]
I had a quick talk with @Wang-Shengtao about this, this is just a consequence of the automatic layout. If you're zoomed in too much or viewing on a smaller screen MkDocs hides the navbar behind a hamburger icon.

**Comment by weinbe58** [comment]
https://configurations.rogerluo.dev/stable/quick-start/
https://docs.pennylane.ai/en/stable/
https://docs.pydantic.dev/latest/

**Comment by Wang-Shengtao** [comment]
> I had a quick talk with @Wang-Shengtao about this, this is just a consequence of the automatic layout. If you're zoomed in too much or viewing on a smaller screen MkDocs hides the navbar behind a hamburger icon.

Yes, that's right. On my desktop, it was fine, but I saw in my laptop just now that the automatic layout folded the top banner. Now, I see it's nicely displayed on my laptop as well. I think it's just the default Zoom size or something. It would be good to make the default Zoom/font size appropriate (for most common computer size) so that the banner can be displayed. On my laptop, I see the tutorial page is still folded, other pages are not.

**Comment by johnzl-777** [comment]
> > I had a quick talk with @Wang-Shengtao about this, this is just a consequence of the automatic layout. If you're zoomed in too much or viewing on a smaller screen MkDocs hides the navbar behind a hamburger icon.
> 
> Yes, that's right. On my desktop, it was fine, but I saw in my laptop just now that the automatic layout folded the top banner. Now, I see it's nicely displayed on my laptop as well. I think it's just the default Zoom size or something. It would be good to make the default Zoom/font size appropriate (for most common computer size) so that the banner can be displayed. On my laptop, I see the tutorial page is still folded, other pages are displaying fine.

I took at the documentation and there doesn't seem to be a way to toggle this feature ): I do notice even on my laptop the zoom in Chrome has to be set to 90% instead of standard 100% to avoid the navbar collapsing into the hamburger icon. @Roger-luo is there a potential thing we could do here without having to fork and create our own theme? Perhaps a CSS override?

**Comment by Roger-luo** [comment]
can you guys put a screenshot of the issue? It's a bit hard to understand what's going on here.

**Comment by johnzl-777** [comment]
Sure! I should preface I did some more digging around and it seems sensitive to how the browser interacts with the detected display dimensions because on one monitor I have to zoom in pretty heavily before the hamburger icon appears while in another it takes much less zooming in.

This is the standard navbar:
<img width="575" alt="Screenshot 2023-10-23 at 4 19 30 PM" src="https://github.com/QuEraComputing/bloqade-python/assets/32608115/dffce9a6-4bd9-4a2d-b8a4-200cfc4f60d5">

And at some point it can collapse to the hamburger icon: 
<img width="895" alt="Screenshot 2023-10-23 at 4 19 46 PM" src="https://github.com/QuEraComputing/bloqade-python/assets/32608115/0e698d40-b115-4157-82e0-d027760bf6ee">

I don't know if there's a way to control at what zoom level/ratio before this collapse happens.

**Comment by Roger-luo** [comment]
this is expected behavior on the small screen. I'm not sure what is the issue here? It should collapse into a hamburger on the small screen.

**Comment by johnzl-777** [comment]
I think @Wang-Shengtao is asking for changing exactly when/what dimensions trigger the collapse into the hamburger menu. But this would be a tricky thing to do because it sounds like he might have different zoom settings from others. I'm inclined to leave things as they are, especially considering all that I can dig up to change this setting requires some lower-level modification.

**Comment by Roger-luo** [comment]
this is adaptive to device size, I'm not sure why we should change this? It's changeable at the CSS level, and we are just using the most standard setting. Can @Wang-Shengtao elaborate on what makes this behave not expected? e.g by steps to reproduce. I'm very confused about this proposal and still don't understand.

Ideally in a separate issue as I don't see this is a blocking issue for this issue, and @johnzl-777 please go ahead and work on other documentation issues, I'll take care of @Wang-Shengtao's comments if necessary.

**Comment by Wang-Shengtao** [comment]
I'm just hoping the default font size can be set so that the top bar doesn't shrink into hamburger menu for the most common screen size (with full screen size, e.g. 16-inch). A few days ago, it appears to be in hamburger menu on my laptop, but now it's no longer the case, which is good. I think the font size got reduced a little bit. For the tutorial page, it's still showing the hamburger menu though - perhaps we can reduce the default Zoom size for the tutorial page a bit as well.
<img width="1112" alt="Screenshot 2023-10-24 at 9 40 20 PM" src="https://github.com/QuEraComputing/bloqade-python/assets/25993974/a1525c71-a310-48a8-a6a8-8a1055bf5437">
<img width="1116" alt="Screenshot 2023-10-24 at 9 40 11 PM" src="https://github.com/QuEraComputing/bloqade-python/assets/25993974/7f437cc3-6e00-4f65-b9bf-d0a2ce006e81">

**Comment by Roger-luo** [comment]
> I'm just hoping the default font size can be set so that the top bar doesn't shrink into hamburger menu for the most common screen size (with full-screen size, e.g. 16-inch).

that shouldn't be the case. This theme is configured to be expanded on most of the screens. The tutorial page is currently being built separately; thus, something is inconsistent between these two websites. Let's move the discussion to that repo, and see if it gets fixed after https://github.com/QuEraComputing/bloqade-python-examples/commit/d8b32ab164afa9257ba510678a5d980ff2b1cd7e
